sct.py

```diff
@@ -322,6 +322,7 @@ def clean_resources(ctx, post_behavior, user, test_id, logdir, dry_run, backend)
     else:
         os.environ['SCT_CLUSTER_BACKEND'] = backend
 
+    os.environ['SCT_TEST_ID'] = test_id[0]
```

Comment:
how did it worked for reserved capacity ? clean-resources never finds it if `SCT_TEST_ID` isn't set...

i.e. you can't clear RC from `clean-resources --user` or `clean-resources --test-id` 

I'll put it until if like:
```
if test_id:
    os.environ['SCT_TEST_ID'] = test_id[0]
```

if we want support of clean-resources by user, it would need a bit more code for lookup hosts by tags, like other AWS resources

Reply:
which does make sense, if we have keep option and such.